### PR TITLE
Add `logger` configuration to output log to file

### PR DESCRIPTION
**Pull request by upscent**
By default, slack-ruby-bot outputs logs to `STDOUT`.
We can silence log by changing log level, but can't change output destination.

So I added logger configuration to set custom logger.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ### 0.10.4 (date)
 
 * Your contribution here.
+* [#148](https://github.com/slack-ruby/slack-ruby-bot/pull/148): Add `logger` configuration to set a custom logger - [@upscent](https://github.com/upscent).
```
Comment (upscent) [inline CHANGELOG.md]: Oh, wrong PR number ... 😇

**Comment by dblock** [comment]
Does it make sense to do this in slack-ruby-client (as well), https://github.com/slack-ruby/slack-ruby-bot/blob/master/lib/slack-ruby-bot/support/loggable.rb#L10 and piggyback on that?

Otherwise perfect. Can you please rebase and I'll merge.

**Comment by upscent** [comment]
@dblock, 
Thank you for your advice to use slack-ruby-client to customize logger.

I rebased latest master!
If it's OK with you, please merge this PR 🙇

**Comment by dblock** [comment]
Merged

**Pull request merged**